octobot_trading/exchanges/__init__.pxd

```diff
@@ -145,6 +145,7 @@ __all__ = [
     "get_default_exchange_type",
     "get_supported_exchange_types",
     "get_exchange_class_from_name",
+    "CCXTExchangeCommons",
```

Comment:
Same here, I think we shouldn't have any CCXT class in the generic part

Reply:
its removed